### DISCUSSION TITLE
Access Azure Active Directory Mgt-Get Response in Template not working

### DISCUSSION THREAD

**Discussion by SoerenOhlsen** [microsoft-search/pnp-modern-search#2391] [Q&A] [answered]
Hello everyone,

we are developing a company people search with the modern search web part. We would like to display additional fields like company name and location field directly in the person element without opening the hovercard (the content is already there by default). Because of the fact, that these fields are not automatically mapped to SP User profile, we tried using the mgt-get element in our template to get the information directly from the Azure Active Directory by Graph API call users/{id}.

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/67585513/192552880-d94abc3d-6e1a-487f-bd34-7837648f1f07.png)
 
In the browser's developer tools, we see that the call against the graph is successfully executed. 
![teams4](https://user-images.githubusercontent.com/67585513/192553097-d83b340b-73e5-4c84-a6e3-0515d7da970b.png)

However, the data cannot be accessed/ displayed. With the following template above, we only get the SPItem back. 
![MicrosoftTeams-image](https://user-images.githubusercontent.com/67585513/192551985-36f17b34-5870-4916-bfbc-f2ee45f0ef35.png)
 
We checked that the graph object should can be accessed in the MGT playground in this way: 
![teams3](https://user-images.githubusercontent.com/67585513/192551977-34c3e4a7-2891-425b-ab79-d7b1c62f14e8.PNG)

What are we doing wrong? 
And how can we access the Graph Object?

Any help is welcome. Thank you in advance.

**Comment by wobba** [comment]
@SoerenOhlsen could this be mgt version specific? As we use v2.3.0 today and v2.6.1 is the latest for mgt.

**Comment by SoerenOhlsen** [comment, marked as answer]
@wobba Thank you for your reply. It's quite possible thats a version-specific issue. Difficult to say as we're using the modern search web part by default and not as a custom development solution. Are there any plans to update mgt version to latest version soon?

**Reply by wobba** [reply]
I'll try. @FranckyC are you aware of any issues if we update the mgt version used?

**Reply by FranckyC** [reply]
This is not a version issue but a syntax issue :D. What you see is actually the Handlebars context. This is because MGT and Handlebars use the same binding syntax with "{{ }}". To bypass that, in your MGT template you need to escape the expression like this:

```
         <li>
        <mgt-get resource="/me" version="beta" scopes="mail.read" max-pages="2">

        	<template>
        		\{{ JSON.stringify(this) }}
        	</template>
        </mgt-get>
```

FYI, the enclosed expression between curly braces is interpreted as plain JavaScript so you even do something like this:
```
\{{ console.log(JSON.stringify(this)) }}
```

**Reply by SoerenOhlsen** [reply]
@FranckyC Thank you very much! It's working now! :)

**Reply by FranckyC** [reply]
You're welcome!
